web/controllers/coherence/session_controller.ex

```diff
@@ -0,0 +1,323 @@
+defmodule CoursePlanner.Coherence.SessionController do
```

Comment:
we shouldn't add this file

I checked it out and the reason you need to add it is that we have the coherence protected routes wrong.

Could you move
```elixir
coherence_routes :protected
```
into it's own block?
```elixir
  scope "/" do
    pipe_through :protected
    coherence_routes :protected
  end
```
then remove this file?